### PR TITLE
Added Canada-Alberta: production, price and exchange (#166)

### PR DESCRIPTION
**Pull request by mlucchini**
Issue:
https://github.com/tmrowco/electricitymap/issues/166

I used current time as it seems to be refreshed constantly and it avoids having to rely on specific HTML parsing but might change that though.

### CODE REVIEW

File: parsers/CA_AB.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+    r = session or requests.session()
+    url = 'http://ets.aeso.ca/ets_web/ip/Market/Reports/CSDReportServlet'
+    response = r.get(url)
+    df_exchanges = pd.read_html(response.text, match='INTERCHANGE', skiprows=0, index_col=0)
+    
+    flows = {
+        'CA-AB->CA-BC': df_exchanges[1][1]['British Columbia'],
+        'CA-AB->CA-SK': df_exchanges[1][1]['Saskatchewan'],
+        'CA-AB->US-MT': df_exchanges[1][1]['Montana']
```
Comment (corradio) [inline parsers/CA_AB.py]: For now we only have US and not US-MT (we don't break down into US states yet).
Can you just replace US-MT with US?

File: parsers/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
-beautifulsoup4==4.5.1
+beautifulsoup4==4.5.3
+html5lib==0.999999999
```
Comment (corradio) [inline parsers/requirements.txt]: Funny version name. Is that intended?

**Comment by mlucchini** [comment]
OK will make the change later. Yeah I think they do use that version as
latest, perhaps they add a 9 per release!?

**Comment by corradio** [comment]
Pretty awesome dude. Thanks! Will test/push to staging tonight.

**Comment by mlucchini** [comment]
No worries!

**Pull request merged**